.github/workflows/docs-preview-pr.yaml

```
@@ -39,7 +40,7 @@ jobs:
               console.log("Request to GitHub API for Pages failed with message: " + err)
             }
             core.setOutput('html_url', '')
-            core.setOutput('branch', '')
+            core.setOutput('branch', '')``
```

Are the backticks supposed to be there? Like I also see these in NVT - but doesn't look quite right to me (and breaks syntax highlighting on my editor =) )
```suggestion
            core.setOutput('branch', '')
```